assignment.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func Parse(text string) (number1 float64, number2 float64, operator string, err error) {
+	splits := strings.Split(text, " ")
```

Comment:
You can use strings.Fields for splitting the string into numbers and operands, it'll also trim the whitespace.